.github/workflows/test-go-generate.yml

```
@@ -0,0 +1,40 @@
+name: Go generate check
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - "release/**"
+  pull_request:
```

```suggestion
  workflow_dispatch:
  push:
    branches:
      - main
      - "release/**"
    paths:
      - "**.go"
      - "**/go.mod"
      - "**/go.sum"
  pull_request:
    paths:
      - "**.go"
      - "**/go.mod"
      - "**/go.sum"
```

This is only needed on changes in Go files, isn't it?

---

Ah, surely this is not the case for the generation of the cilium helm chart...